src/protocol/PasLS.Symbols.pas

```diff
@@ -714,15 +713,15 @@ procedure AddField(var Source: String; Value: Integer; Terminate: Boolean = true
 end;
 
 const
-  SYMBOL_ENTRY_NAME = 0;
-  SYMBOL_ENTRY_PATH = 1;
+
+//  SYMBOL_ENTRY_NAME = 0;
+//  SYMBOL_ENTRY_PATH = 1;
   SYMBOL_ENTRY_JSON = 2;
```

Comment:
Should this be an enum then maybe? These are the column indexes for the DB so it's kind of annoying to have to remove them just because the compiler is making noise.

Reply:
An enum is fine. I'll change accordingly